operator/pkg/helmreconciler/prune.go

```diff
@@ -177,6 +179,34 @@ func (h *HelmReconciler) DeleteObjectsList(objectsList []*unstructured.Unstructu
 	return errs.ToError()
 }
 
+// RemoveIstioCaRootCert remove the istio ca cert made by istio namespace controller
+func (h *HelmReconciler) RemoveIstioCaRootCert() error {
```

Comment:
You may have multiple Istio installs, this is removing it from ALL of them

Reply:
yes,it will remove all configmap *istio-ca-root-cert*. but I have some questions about it .
First:
It will leave the configmap *istio-ca-root-cert* in namespace if we run *istioctl x uninstall --purge*.
```
The optional --purge flag will remove all Istio resources, including cluster-scoped resources that may be shared with other Istio control planes.
``` 
This is not the same as the description.
Second:
Even if I delete the cm, if there are other control plane, the namespace controller will automatically create it immediately.
Thrid:
I tested and found that if a custom certificate is used and cm exists in the current namespace, the application under this namespace will fail to start.
Is there any other better way to solve this problem? thanks. @howardjohn